### PR TITLE
Syntax change for CSV options in copy from and external tables

### PR DESCRIPTION
**Pull request by sgilic**
enable new fields for csv external table and copy from csv file.

PRD: https://docs.google.com/document/d/1HyAsvRb0S8iv5bjIEAW424fCGyB3o-O3u9lITizw-OQ/edit?ts=5b83bb25

### CODE REVIEW

**Comment by sgilic** [comment]
@soupi please review my code

File: hssqlppp/src/Database/HsSqlPpp/Pretty.lhs [hunk 1]
```diff
@@ -1118,28 +1124,85 @@ syntax maybe should error instead of silently breaking
...
->           <+> quotes (text path)
->           <+> ppDelimiter
->           <+> ppRecord
->         
+>   EtCsvOptions csvOpts@(NewCsvOptions{csvFilePath=path}) ->
+>     text "path" <+> quotes (text path) <+> ppCsvOpts csvOpts
+
+> ppCsvOpts :: CsvOptions -> Doc
+> ppCsvOpts NewCsvOptions
+>   { csvFilePath = path
```
Comment (soupi) [inline hssqlppp/src/Database/HsSqlPpp/Pretty.lhs:1143]: Remind me to show you how this can be written more easily.

**Comment by soupi** [comment]
Looks good. Thanks!

**Comment by sgilic** [comment]
Thanks a lot for the review and the quick response, much appreciated!
Have a great vacation :)

**Pull request merged**